tests/run_tests.sh

```diff
@@ -570,6 +570,10 @@ dtc_tests () {
     run_dtc_test -I dts -O dtb -o dtc_sized_cells.test.dtb "$SRCDIR/sized_cells.dts"
     run_test sized_cells dtc_sized_cells.test.dtb
 
+    run_dtc_test -I dts -O dts -o over-determined-once.dts "$SRCDIR/over-determined.dts"
+    run_dtc_test -I dts -O dts -o over-determined-twice.dts "over-determined-once.dts"
+    run_wrap_test cmp over-determined-once.dts over-determined-twice.dts
```

Comment:
I generally mistrust directly comparing dts files, since it can pick up meaningless format differences.  But I guess both files you're comparing here are generated by the same dtc version, so it's probably ok.